src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1222,8 +1222,8 @@ public static partial class Path
         public static string Combine(string path1, string path2, string path3) { throw null; }
         public static string Combine(string path1, string path2, string path3, string path4) { throw null; }
         public static string Combine(params string[] paths) { throw null; }
-        public static string TrimEndingDirectorySeparator(string path) { throw null; }
         public static ReadOnlySpan<char> TrimEndingDirectorySeparator(ReadOnlySpan<char> path) { throw null; }
+        public static string TrimEndingDirectorySeparator(string path) { throw null; }
```

Comment:
@MarcoRossignoli i think dan meant to have these new trim apis in alpha order w.r.t existing functions i.e. after join

Reply:
ahh...ok

Reply:
done